### PR TITLE
startup_regtest.sh: add connect helper and experimental-offers

### PR DESCRIPTION
**Pull request by jb55**
Here are a few additions to `startup_regtest.sh` to make it easier to connect nodes via a new `connect` command.

Also add `experimental-offers` to the node configs for testing offers.

### CODE REVIEW

**Comment by jb55** [comment]
On Fri, Mar 25, 2022 at 01:23:30PM -0700, Vincenzo Palazzo wrote:
>Maybe is a better naming convention?

I just wanted to make it clear that it takes node numbers

File: contrib/startup_regtest.sh [hunk 1]
```diff
@@ -216,3 +213,12 @@ stop_elem() {
 	unset LN_NODES
 	unalias et-cli
 }
+
+connect() {
+	if [ -z $1 ] || [ -z $2 ]; then
+		printf "usage: connect 1 2\n"
+	else
+		to=$($LCLI --lightning-dir=/tmp/l$2-$network getinfo | jq -r '"\(.id)@\(.binding[0].address):\(.binding[0].port)"')
```
Comment (rustyrussell) [inline contrib/startup_regtest.sh:221]: Hmm, let's not rely on jq?  Maybe:

```
lightning-cli -F getinfo | grep '^\(id\|address\[0\]\.\(address\|port\)\)' | cut -d= -f2- | tr '\n' ' ' | (read ID ADDR PORT; echo $ID@${ADDR}:$PORT)
```

**Comment by ddustin** [comment]
This PR may be relevant https://github.com/ElementsProject/lightning/pull/5062 as it builds two nodes and connects them as well

**Comment by niftynei** [comment]
I want the `experimental-offers` patch! We're looking to get a RC out this week. If the command in #5092 solves this, maybe we just make this the first patch?

File: contrib/startup_regtest.sh (hunk 1 above)
Comment (jb55) [reply contrib/startup_regtest.sh]: done

**Comment by jb55** [comment]
I think I have appeased the shellcheck gods to a sufficient degree for this change to be acceptable.

**Review by rustyrussell — APPROVED** [review]
Ack 8e01c925e272bef3dae4c86a9a1b351b504d78b4

**Pull request merged**